hr_timesheet_project_access_restriction/security/hr_timesheet_project_access_restriction_security.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <!-- Overrride standard record rule to allow simple employee to access
+        on project in which they are member-->
+        <record model="ir.rule" id="project.project_public_members_rule">
+            <field name="name">Project: employees: public, employees, followers, members or project responsible</field>
+            <field name="domain_force">['|', '|', '|',
+                                            ('privacy_visibility', 'in', ['public', 'employees']),
+                                            ('message_follower_ids', 'in', [user.partner_id.id]),
+                                            ('user_id', '=', user.id),
+                                            ('members', 'in', [user.id])
+                                        ]</field>
+        </record>
+        <!-- Overrride standard record rule to allow simple employee to access
+        on analytic account in which they are member on the related project-->
+        <record id="analytic_project_public_members_rule" model="ir.rule">
+            <field name="name">Analytic Account - Project: employees: public, employees, followers, members or project responsible</field>
+            <field name="domain_force">['|', '|', '|',
+                                            ('project_ids.privacy_visibility', 'in', ['public', 'employees']),
+                                            ('project_ids.message_follower_ids', 'in', [user.partner_id.id]),
```

Comment:
What's the difference between followers and members? Do we have to grant access to the followers on the analytic accounts?